### PR TITLE
Roberto Rodriguez

### PR DESCRIPTION
**Pull request by robertorodriguez12**
Here is my prework

### CODE REVIEW

File: day_6/exercises/burrito.rb [hunk 1]
```diff
@@ -17,3 +18,13 @@ def initialize(protein, base, toppings)
...
+
+#Add topping
+dinner.toppings << 'Jalapenos'
+p dinner.toppings
+# remove topping
+dinner.toppings.delete_at(2)
+p dinner.toppings
+# change_protein
+dinner.protein = "Steak"
+p dinner.protein
```
Comment (ap2322) [inline day_6/exercises/burrito.rb:30]: Clever use of the `attr_accessor` methods combined with Ruby methods to achieve the state change to `dinner`. Try re-writing these as methods within your `Burrito` class and then calling the methods to achieve the same output.

**Review by ap2322 — COMMENTED** [review]
@robertorodriguez12 nice work getting this done! I left a couple comments that could be good things to go back over in preparation for day 1.

File: day_7/high_level_notes.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  3.1 In your terminal, open your code editor application ```For this example, we are using ATOM```
+    3.1.1 To open Atom, if you have the shell commands installed, type ```atom .``` in to your terminal and press enter. This will open Atom
+  3.2 In Atom, open your caesar_cipher.rb file
+  3.3 write your code
+    3.3.1 If you do not know how to write the code to create a Caesar cipher then conduct a Google search on it
+      3.3.1.1 Open your web browser
+      3.3.1.2 navigate to the Google homepage by typing www.google.com and pressing enter
+      3.3.1.3 search for how to create a Caesar cipher in Ruby
+      3.3.1.4 choose the search result that best shows you how to write this code
+    3.3.2 Once you finish writing your code, save it
```
[the remark this answers is not in the document]
Comment (robertorodriguez12) [reply day_7/high_level_notes.md:22]: So, when it comes to this portion of the Capstone, should I research different methods of completing this and then submitting it again???

File: day_6/exercises/burrito.rb [hunk 3]
```diff
@@ -1,11 +1,12 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
```
[the remark this answers is not in the document]
Comment (robertorodriguez12) [reply day_6/exercises/burrito.rb:5]: def add(toppings)
  end

# 2
  def remove(toppings)
  end

# 3
  def protein(protein)
    @protein = protein
  end

Would this work better?

[the remark this answers is not in the document]
Comment (ap2322) [reply day_6/exercises/burrito.rb:5]: That is closer to using defined methods. Based on the instructions you would define methods `add_topping`, `remove_topping`, and `change_protein`. It's up to you if you want to pass in arguments like you showed on `def protein(protein)`.

These methods would execute the code that you currently are using outside of the class `Burrito`. Then, after you initialize your `dinner` instance, you can call the methods.

Does that make sense?

File: day_7/high_level_notes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ap2322) [reply day_7/high_level_notes.md:22]: You don't need to re-submit your capstone. Sorry for the confusion. These are suggestions for you to work through on your own repo in preparation for Mod 1.

**Comment by robertorodriguez12** [comment]
Ahhhh ok.  Yeah it makes sense now.  I'm sorry, I'm just trying to figure
things out still :-).

**Pull request closed without merging**